portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,190 @@
+@import url('https://fonts.googleapis.com/css?family=Major Mono Display');
+@import url('https://fonts.googleapis.com/css?family=Codystar');
+@import url('https://fonts.googleapis.com/css?family=Nothing You Could Do');
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin-left: 15%;
+  margin-right: 15%;
 }
 
 #greeting-container {
   margin-top: 20px;
+  
+}
+
+#TechCorner {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+#Contact {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+
+img.image {
+    float: right;
+    padding: 20px;
+}
+
+p.intro, div a{
+    width: 50%;
+    white-space: pre-wrap;  
+    word-wrap: break-word;
+
```

Comment:
Don't forget to use the formatter to format all these files! If the IDE setup is blocking you, you can do formatting in a different PR. Using a formatter would also help automatically fix some of the issues in Alfred's comments!

Reply:
will do:)